gitgud/test_user.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from gitgud.__main__ import GitGud
+
+def test_assert_initialized():
+    pass
+def test_handle_commit():
+    pass
+def test_handle_contrib():
+    pass
+def test_handle_debug():
+    pass
+def test_handle_explain():
```

Comment:
Ideally, we test explain/goal/status on all levels and at all stages during the solving, but to keep things lightweight, let's maybe just do it for one level as a sanity check?

We could run the commands on all levels, but that is more time-consuming. What are your thoughts?